.github/workflows/SDK-Suppressions-Label.yaml

```
@@ -0,0 +1,114 @@
+name: SDK Suppression Label
+
+on:
+  pull_request:
+    branches:
+      - main
+      - RPSaaSMaster
+      - release*
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+  workflow_dispatch:
+  
+jobs:
+  process-sdk-suppressions-labels:
+    name: Sdk Suppression Labeling
```

```suggestion
    name: SDK Suppressions
```

Single-job workflows should use same name for job and workflow.